src/main/java/com/redhat/devtools/intellij/quarkus/module/QuarkusCodeEndpointChooserStep.java

```diff
@@ -115,34 +147,117 @@ private void updateCustomUrl() {
         this.endpointURL.getTextEditor().setEditable(custom);
         this.endpointURL.setEnabled(custom);
         this.customUrlWithBrowseButton.setButtonEnabled(custom);
+        if (loadingRequest != null) {
```

Comment:
`if (loadingRequest != null && !loadingRequest.isDone()) {`

Reply:
doesn't matter, cancel is a no-op on completed futures